resources/views/components/button-svg-overlay-dark.blade.php

```diff
@@ -0,0 +1,12 @@
+{{--
+    $button => array // [['title', 'link', 'relative_url', 'secondary_relative_url']]
+--}}
+
+@if(!empty($button['link']) && !empty($button['relative_url']) && !empty($button['secondary_relative_url']))
+    <a href="{{ $button['link'] }}" class="block min-w-full relative rounded bg-cover bg-green-darkest" style="padding-top: 36.39%; background-image: url('{{ $button['relative_url'] }}'); ">
+        <div class="absolute pin p-4 rounded bg-green-darkest opacity-65"></div>
+        <div class="absolute min-w-full pin rounded">
+            <img src="{{ $button['secondary_relative_url']}}">
```

Comment:
This also needs an `alt` attribute with the text that should be read to a screen reader. Likely the `title`.